### ISSUE TITLE
Olives - Vat Recipe of some kind

### ISSUE THREAD

**Issue by Darkosto** [Darkosto/SevTech-Ages#355]
Currently Olives into olive oil is useless, maybe we can add a recipe using IE or another liquid processing to give it a use.

**Comment by EhGneissGuy** [comment]
Just made up a big batch only to find out that there's no use for it. Glad to see that this is on the docket!

**Comment by Gammelkebab** [comment]
Yeah as far as I can see olives in general are baggage. You eat them - you get very little hunger restored and nausea. You stomp them? You get olive oil which gives in a sum even less food and still makes you nauseous. So no use for food and no recipes either.

**Comment by Kiefy** [comment]
At least disable the Oil recipe so we don't waste our time making stacks of it only to realise it's been disabled.

Yes, I'm salty.
